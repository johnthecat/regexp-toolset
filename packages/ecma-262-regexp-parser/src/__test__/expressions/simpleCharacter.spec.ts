import { expect, describe, it } from 'vitest';
import { parseRegexp } from '../../index.js';

/**
 * @see parseSimpleChar
 */
describe('Simple char', () => {
  it('should correctly parse simple value', () => {
    expect(parseRegexp(/a/)).toMatchSnapshot();
  });
});
