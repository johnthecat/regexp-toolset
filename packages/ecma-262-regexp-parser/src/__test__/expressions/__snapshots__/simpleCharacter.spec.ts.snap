// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Simple char > should correctly parse simple value 1`] = `
{
  "body": {
    "charCode": 97,
    "end": 1,
    "kind": "Char",
    "start": 1,
    "type": "Simple",
    "value": "a",
  },
  "end": 2,
  "flags": "",
  "kind": "Regexp",
  "start": 0,
}
`;
